docs/asciidoc/more_background_on_compact_identifiers_content.adoc

```diff
@@ -0,0 +1,145 @@
+== About
+
+This document contains more examples of resolving compact identifier-based DRS URIs than we could fit in the DRS specification or appendix.  It's provided here for your reference as a supplement to the specification.
+
+== Hostname-Based URIs
```

Comment:
Did you mean to include this info on Hostname-Based URIs in the new document? (I think there may be other unintended content, but will wait until you've had a chance to respond before looking more closely.)

Reply:
Yep, I removed hostnames from this doc and just added a note that accessions from compact identifiers may need encoding before being used as a DRS ID in DRS API calls given the characters allowed in DRS IDs.